src/org/zaproxy/zap/utils/HirshbergMatcher.java

```diff
@@ -123,23 +105,14 @@ private static void algC(StringBuilder sb, int m, int n, String a, String b) {
     /**
      * This method takes a string as input reverses it and returns the result
      * 
-     * @param in
-     * @return
+     * @param in the string to be reversed
+     * @return the reversed string
      */
     private static String reverseString(String in) {
         StringBuilder out = new StringBuilder(in).reverse();
         return out.toString();
     }
 
-    /**
-     * This method finds the index of the maximum sum of L1 and L2, as described
-     * by Hirschberg
```

Comment:
Is it more clear with that? What is L1, L2 and n?

I was going to document these methods properly but didn't find the paper with the description of the algorithm.